frontend/src/views/UserDetails/index.vue

```diff
@@ -0,0 +1,80 @@
+<script setup lang="ts">
+import { useRoute } from 'vue-router';
+import { hashRouter } from '@/router';
+import Button from 'primevue/button';
+import UserSummaryCard from '@/components/UserSummaryCard';
+import PageTitle from '@/components/common/PageTitle.vue';
+import UserPermissionHistoryTable from '@/components/UserPermissionHistoryTable';
+import BreadCrumbs from '@/components/BreadCrumbs';
+import type { CrumbType } from '@/components/BreadCrumbs/definitions';
+
+const route = useRoute();
+
+// Access the path parameters
+const userId = route.params.userId as string | undefined;
+const applicationId = route.params.applicationId as string | undefined;
+
+if (!userId || !applicationId) {
+  console.warn("Missing required path params");
+  hashRouter.push('/');
+}
```

Comment:
This check is a good practice because it adds an extra layer of safety. While the parent component is supposed to provide the correct IDs, unexpected situations can occur (e.g., bad navigation or manual URL changes by users). This ensures that the app gracefully handles any missing data and prevents errors or unexpected behavior.